osu.Game.Rulesets.Osu/Mods/OsuModIncognito.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace osu.Game.Rulesets.Osu.Mods
 {
-    internal class OsuModVisualAdjusts : ModVisualAdjusts, IApplicableToDrawableRuleset<OsuHitObject>, IApplicableToDrawableHitObject
+    internal class OsuModIncognito : ModIncognito, IApplicableToDrawableRuleset<OsuHitObject>, IApplicableToDrawableHitObject
     {
         [SettingSource("Disable follow points", "No more hints for where to follow...")]
```

Comment:
Not a fan of this description it feels somewhat clumsily formulated. 

I'd opt for either something tongue in cheek... say... " The lines... where are they?" or " disconnected"

Given the main setting name is already very clear, so having some fun with the description seems appropriate. 

Even if that isn't the desired path id still rephrase it as something like " No hints on where to go next.." or " Find your way from circle to circle unaided"

Reply:
Yeah, i was waiting for someone to point out this it seemed odd to me too, is just that i couldn't think of a way to rephrase it.
I will go with "The lines... where are they?'